### PR TITLE
Notes on cryocloud docker image selection and use

### PR DESCRIPTION
**Pull request by scottyhq**
Discussed originally on slack. Not sure of the best place in the table of contents for this...

### CODE REVIEW

File: book/how_tos/docker.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+```
+
+Note that the tag `17aa6e2ed1a6` corresponds to a commit on GitHub, so we can see recover the exact environment definition at any point in time (https://github.com/CryoInTheCloud/hub-image/tree/17aa6e2ed1a6)
+
+
+## Use a previous environment on CryoCloud
+
+If you want to keep using an older environment definition on the JupyterHub login page for the 'Image' dropdown Menu select `Other`. Then put the URL into the 'Custom image' box (quay.io/cryointhecloud/cryo-hub-image:17aa6e2ed1a6).
+
+[Insert Screenshot](screenshot)
```
Comment (scottyhq) [inline book/how_tos/docker.md:24]: TODO: this is not currently a Hub feature but could be implemented https://infrastructure.2i2c.org/howto/features/allow-unlisted-profile-choice/#allow-users-to-setup-custom-free-form-user-profile-choices

**Comment by tsnow03** [comment]
We will have a Must Know document of a sort up on the book by Monday hopefully. We will tag you on it so you can add a little to it.

**Comment by tsnow03** [comment]
@scottyhq Feel free to add this information to the [CryoCloud Best Practices](https://github.com/CryoInTheCloud/CryoCloudWebsite/blob/main/book/content/hub_best_practices.md) page, which is posted now. Also feel free to add anything else!

**Comment by jdmillstein** [comment]
this information has been added to https://book.cryointhecloud.com/content/hub_best_practices.html

**Pull request closed without merging**